mitodl/hugo-course-publisher
# travis build & deploy to s3 on release

#### Pre-Flight checklist


#### What are the relevant tickets?
https://github.com/mitodl/hugo-course-publisher/issues/184

#### What's this PR do?
This PR adds a build script to the project called `s3_upload.js`.  This script handles clearing out the RC / Prod site S3 bucket and replacing its contents with the output of the full site build.

#### How should this be manually tested?
The `release` branch has been created and is in sync with this branch, `cg/travis-build`.  In Travis, the build on the release branch should trigger a conditional `deploy` job which wipes the content of the S3 bucket configured at `AWS_STORAGE_BUCKET_NAME` and uploads the new build.  Ensure that the build uploaded properly, all the files are in place and the site is browsable at https://ocw-beta.odl.mit.edu.
